### PR TITLE
Make adjustments for new pdf theme

### LINKED ISSUE #26: Make adjustments for new PDF theme
To be able to use the new theme, we need to:
- [x] Fix the fonts in the excel file
- [x] export new example images
- [x] upgrade asciidoctorJ-PDF to ~v1.5.0.beta1~ v1.5.0.beta.2. It is not yet released, but is required for an important bugfix (see https://github.com/asciidoctor/asciidoctor-pdf/issues/1087 and https://github.com/asciidoctor/asciidoctor-pdf/issues/1107)
- [x] update the submodule once https://github.com/isaqb-org/pdf-theme/issues/15 is complete

### PR DESCRIPTION
**Pull request by programming-wolf**
Please wait for the designer to review the final document (read: my explicit permission) before merging this PR. 

Close #26

### CODE REVIEW

**Comment by programming-wolf** [comment]
The designer approved all changes. 🙂

File: build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ plugins {
 
 ext {
   currentDate = new Date().format("d. MMM yyyy")
-  project.version = "0.1.0 DRAFT ($currentDate)"
+  project.version = project.file("./document.version").text
```
Comment (larsrh) [inline build.gradle:19]: Should probably add a fallback here if that file doesn't exist.

File: build.gradle [hunk 2]
```diff
@@ -40,9 +40,14 @@ class RenderCurriculumTask extends AsciidoctorTask {
     outputDir = new File("./build/")
     separateOutputDirs = false
     backends 'pdf', 'html5'
+
+    def fileVersion = project.file("./document.version").text + "-" + language
```
Comment (larsrh) [inline build.gradle]: … and pull this out into a function (if Gradle supports that).

File: build.gradle (hunk 1 above)
Comment (programming-wolf) [reply build.gradle:19]: No. I want the build script to fail hard if it is not there, because it is a mandatory file.

**Review by larsrh — APPROVED** [review]

File: build.gradle (hunk 2 above)
Comment (programming-wolf) [reply build.gradle]: Well, I now use `project.version` instead, which is assigned with the same value in line 19.

**Pull request merged**